### PR TITLE
dbcsr: add new package

### PR DESCRIPTION
**Pull request by davydden**
@dev-zero @alazzaro FYI. ~~I would appreciate if you could confirm the manual installation phase.~~

### CODE REVIEW

**Comment by tgamblin** [comment]
Needs new license headers now that the relicense is done :)

**Comment by scheibelp** [comment]
> Needs new license headers now that the relicense is done

See https://github.com/spack/spack/pull/9450#issue-221027331 for an example of what that looks like

**Comment by davydden** [comment]
> Needs new license headers now that the relicense is done :)

done.

File: var/spack/repos/builtin/packages/dbcsr/package.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/dbcsr/package.py]: @tgamblin does it matter whether there's an extra `#` here?

@davydden to make this consistent the license header would be:

```
# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
# Spack Project Developers. See the top-level COPYRIGHT file for details.
# 
# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```

Comment (davydden) [reply var/spack/repos/builtin/packages/dbcsr/package.py]: i updated the header.

File: var/spack/repos/builtin/packages/dbcsr/package.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/dbcsr/package.py]: @davydden Apparently flake has become more strict: it is complaining about this unused import.

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**